pytorch_lightning/metrics/classification/precision_recall.py

```diff
@@ -218,3 +237,102 @@ def compute(self):
             return self.true_positives.sum().float() / (self.actual_positives.sum() + METRIC_EPS)
         elif self.average == 'macro':
             return (self.true_positives.float() / (self.actual_positives + METRIC_EPS)).mean()
+
+
+Sensitivity = Recall
+
+
+class Specificity(Metric):
```

Comment:
can we define it as functional and then add class as a wrapper?

Reply:
I could add the functional. 

While for the modulized parts, I followed the same style as Recall and Precision. If a wrapper shall be used, should I update those functions as well?

Reply:
@shijianjian then intention is that all class metrics should call their functional counterpart, but recall and precision have still not been updated to this format. Please see how the `ConfusionMatrix` metric is organised :]